src/modules/common/components/Error.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import FontAwesome from 'react-fontawesome'
+
+import style from './Error.css'
+import { Link } from 'redux-little-router'
+
+export class Error extends React.Component {
```

Comment:
I renamed this to Failure because of a conflict with an ES6 class.

Reply:
I don't like the name, would rather call it ErrorComponent or something like that

Reply:
pls any argument? :P
ErrorComponent doesn't follow our implicit naming convention. No other component has Component in its name.

Reply:
the argument is that I don't like the name Failure and it's always Error :P

Reply:
but yeah sure both soultions aren't very good